### PR TITLE
Feature normalization PR

### PR DESCRIPTION
**Pull request by Phaneron97**
added normalization, 3 examples images aren't converted correctly yet because grayscale first needs to be applied.

### CODE REVIEW

**Comment by Phaneron97** [comment]
* I also made some MWE files to test with which are now unnecessary.
* Deleted readme.txt as we now have readme.md for more readability.

**Review by Bastiaan27 — APPROVED** [review]
lgtm

**Comment by Bastiaan27** [comment]
jojo-go-ahead

File: code/data_analysis/normalization_converter.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        plt.subplot(3, 4, 4*i)
+        plt.hist(normalized_img.ravel(), 256, [0, 256])
+        plt.title('Histogram {}'.format(idx+1))
+        plt.axvline(x=np.mean(normalized_img), color='r', linestyle='dashed', linewidth=2)
+        plt.xlabel('Intensity')
+        plt.ylabel('Frequency')
+    
+    plt.show()
+
+def get_images_in_folder(folder_path):
```
Comment (Sara-Eftekhar-Azam) [inline code/data_analysis/normalization_converter.py:55]: This function is good if the number of images is not very big. If there are many images, the computer will run out of memory. The whole dataset might be too big to fit into the memory. The small version probably fit.

**Review by Sara-Eftekhar-Azam — APPROVED** [review]

File: code/data_analysis/normalization_converter.py (hunk 1 above)
Comment (Bastiaan27) [reply code/data_analysis/normalization_converter.py:55]: The dataset is 4GB which should be able to fit in a pc with 16GB of RAM, or am I wrong?

**Comment by Phaneron97** [comment]
I'm merging this into development as this is approved.

**Pull request merged**